app/css/vendors/ip-camera.css

```diff
@@ -0,0 +1,77 @@
+.camera-view {
```

Comment:
__question:__ is ```vendor``` folder for service specific styles or it's more about _real_ vendors/producers?